salt/annotations/config/home-deployuser-annotations-docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+
+services:
+    fpm:
+        image: "elifesciences/annotations_fpm:${IMAGE_TAG}"
+        volumes:
+            - /srv/annotations/config.php:/srv/annotations/config.php
```

Comment:
The /srv/annotations repository with the provisioned configurations is still present on the host, and these files are mounted inside the container.